transform/mattermost-analytics/models/intermediate/data_eng/signup/int_portal_prod_signups_aggregated_to_users.sql

```diff
@@ -24,13 +24,9 @@ WITH identifies as (
 ), signups as(
     SELECT
         identifies.portal_customer_id,
-        MAX(CASE WHEN pageviews.event_table = 'pageview_verify_email' THEN true ELSE false END) AS verify_email,
-        MAX(CASE WHEN pageviews.event_table = 'pageview_create_workspace' THEN true ELSE false END) AS create_workspace,
-        MAX(CASE WHEN pageviews.event_table = 'pageview_cloud_landing_page' THEN true ELSE false END) AS cloud_landing_page,
-        MAX(CASE WHEN pageviews.event_table = 'pageview_create_signup_password' THEN true ELSE false END) AS create_signup_password,
-        MAX(CASE WHEN pageviews.event_table = 'pageview_getting_started' THEN true ELSE false END) AS pageview_getting_started,
-        MAX(CASE WHEN pageviews.event_table = 'pageview_true_up_review_cws' THEN true ELSE false END) AS pageview_true_up_review_cws,
-        MAX(received_at) as date_updated
+        true AS account_created,
+        MAX(CASE WHEN pageviews.event_table = 'pageview_create_workspace' THEN true ELSE false END) AS verify_email,
```

Comment:
@munish7771 , is this change because we cannot get (or we don't capture them for some reason)  the other 'pageview_*` entries from the event_table?

Reply:
We don't capture them. Only one of the `pageview*` tables can be used to get the correct stage only for email_verified. I have added it as comment int the int_ model.

Reply:
Does that mean that the code in the currently used model was not functional? cc: @kevin-fayle-mm